README.rst

```diff
@@ -64,6 +64,19 @@ variables as necessary. Once done, source the file to get all of the variables i
 
     open vnc://localhost:35900
 
+
+Creating a test course on stage
+-------------------------------
+
+By default, the tests in this repo run against our staging server. If you plan to keep this as the target environment, you must create your own course on stage (this requires an account with staff access). Using your own course will prevent collisions with other developers and ensure that your test results are accurate.
+
+To create the course:
+
+1. Log into studio.edx.org with staff credentials
+2. Create a new course, giving it a unique combination of Organization/ Course Number/ Course Run. The Course Name won't matter here, as it will be overwritten to "E2E Test Course" by the next step.
+3. Go to the course import page and upload the ``course.tar.gz`` file found in the ``upload_files`` directory of this repo.
+
```

Comment:
Since they have added unique org/run/number, they will need to reset their environment variables no? We should call this out explicitly.

Reply:
I think so. It would be easy for someone who is new to the repo to overlook the fact that they need to sync these values up.